### PR TITLE
[WIP]: Please Use Native Markdown

### PR DESCRIPTION
**Pull request by mr-mustash**
Everything you were trying to do in HTML can be accomplished with native markdown. This PR is a full refactor of the boyfriend list.

Edit: Please allow me a few minutes after the current batch of PRs to do the refactor.

### CODE REVIEW

**Comment by Warcop** [comment]
+1

**Comment by igorbrigadir** [comment]
+1

LGTM!

This will break existing PRs and require extra changes, so it might be better to wait to merge others first, and then correct in here

**Comment by mr-mustash** [comment]
Please don't merge yet. I will refactor the rest of the list after this round of PRs is reviewed.

**Comment by Warcop** [comment]
@mr-mustash suggesting rename until later for conflict resolution

**Comment by endingwithali** [comment]
merge conflict pls fix

**Comment by Warcop** [comment]
sync'd approved

File: README.md [hunk 1]
```diff
@@ -1,61 +1,59 @@
-# The Boyfriend List
 
-Submit a PR with a link to your twitch or github handle to join the boyfriend list. You must have a number next to your name. PR Number = Boyfriend Number. No Deleting other boyfriends. 
+yfriend List
```
Comment (endingwithali) [inline README.md]: ruined first line :*(

**Review by endingwithali — APPROVED** [review]

**Pull request merged**